vsutil/func.py

```diff
@@ -1,5 +1,5 @@
 """
-_
+Non-vapoursynth related functions and decorators.
```

Comment:
Since both of our current decorators are for checking clip properties, I’d argue that “non-vapoursynth” isn’t really accurate.

Reply:
Hmm, yea was wondering how to word this, the non-vapoursynth here applies to the functions, the decorators are the only thing on the file that use the vapoursynth module. What would be a better name here you think?

Reply:
Maybe just change the order to make it clear that non-vapoursynth only refers to the functions.
“Decorators and non-vapoursynth-related (should that be double hyphenated?) functions”

Reply:
Not sure about the double hyphen but I think the point gets across either way.